### PR TITLE
Update config.py

### PR DESCRIPTION
**Pull request by jackpertschuk**

### CODE REVIEW

File: pinecone/config.py [hunk 1]
```diff
@@ -137,8 +137,8 @@ def _preprocess_and_validate_config(self, config: dict) -> dict:
         and raises ValueError in case an invalid value was specified.
         """
         # general preprocessing and filtering
-        result = {k.strip(): v.strip() for k, v in config.items() if v is not None}
-        result = {k: v for k, v in result.items() if k in ConfigBase._fields}
+#         result = {k.strip(): v.strip() for k, v in config.items() if v is not None}
```
Comment (benjaminran) [inline pinecone/config.py:140]: Why comment out this line?

Comment (jackpertschuk) [reply pinecone/config.py:140]: it throws an error, because v for openapi_config is of type OpenAPI "Configuration", which has no `strip()` method

Comment (jackpertschuk) [reply pinecone/config.py:140]: we can fix this however you want

Comment (benjaminran) [reply pinecone/config.py:140]: Ah, makes sense

**Review by benjaminran — APPROVED** [review]

**Pull request merged**